src/utils/looper.ts

```diff
@@ -23,21 +23,58 @@
  **********************************************************************************/
 
 import { Config } from '../interfaces/config'
+import { Probe } from '../interfaces/probe'
 import { doProbe } from '../components/http-probe'
+import { log } from '../utils/log'
 
 const MILLISECONDS = 1000
+const DEFAULT_THRESHOLD = 5
+
+function sanitizeProbe(probe: Probe, index: number): Probe {
+  const { name, incidentThreshold, recoveryThreshold, alerts } = probe
+  probe.id = `${index}`
+
+  if (!name) {
+    probe.name = `monika_${probe.id}`
```

Comment:
IIRC there is a key called `name` in probe. I think it should use the probe name first, if there isn't any, fallback to `monika_${probe.id}`

Reply:
Please create issue for this, because i only move existing code, without add or change it's behavior or output.